components/feedstock-card.js

```diff
@@ -72,10 +74,22 @@ export const FeedstockCard = ({ feedstock }) => {
   const id = title.toLowerCase().replace(/\s+/g, '-') // Convert title to id
 
   useEffect(() => {
-    // Check if the current URL hash matches the id of this card
-    if (window.location.hash === `#${id}`) {
-      document.getElementById(id)?.scrollIntoView({ behavior: 'smooth' })
+    const checkIfSelected = () => {
+      if (window.location.hash === `#${id}`) {
+        setIsSelected(true)
+        cardRef.current?.scrollIntoView({ behavior: 'smooth' })
```

Comment:
I think this manual scroll behavior is competing with the default scroll-to behavior that you get by rendering an anchor tag with a hashed route, and this is causing the interaction to feel really stuttered.

What about removing this and just relying on the browser scroll-to?

Reply:
i might be missing something, but the last time I tried to manually scroll, it only worked for hash changes after the initial page load. for instance, when you use this link: [https://catalog.leap.columbia.edu/#hadley-centre-sea-ice-and-sea-surface-temperature-(hadisst)](https://catalog.leap.columbia.edu/#hadley-centre-sea-ice-and-sea-surface-temperature-(hadisst)), without the manual scroll, the page loads but doesn't scroll to the card.

i'm curious to learn what changes are necessary for this hook's implementation to work for initial page load as well. 



```jsx
  useEffect(() => {
    const checkIfSelected = () => {
      if (window.location.hash === `#${id}`) {
        setIsSelected(true)
        setTimeout(() => setIsSelected(false), 3000)
      } else {
        setIsSelected(false)
      }
    }

    checkIfSelected()
    window.addEventListener('hashchange', checkIfSelected)

    return () => window.removeEventListener('hashchange', checkIfSelected)
  }, [id])
```

Reply:
Ah I see what you mean (e.g., using [this link](https://leap-data-catalog-4rmnbdv9k-carbonplan.vercel.app/#cesm-atm-025deg)).

Maybe possible to do a one-time check on page load and manually scroll into view in that case? And then on subsequent `hashchange`s, just rely on the default scroll-to behavior.

Reply:
great... just pushed changes that handle both scenarios